cppcache/include/geode/Struct.hpp

```diff
@@ -154,17 +140,15 @@ class APACHE_GEODE_EXPORT Struct : public Serializable {
   }
 
  private:
+  Struct() = default;
```

Comment:
Is this used by friend classes?  If not we could make it public and delete...

Reply:
Pure virtual class that can only be instantiated by it's sub-classes. Making it private doesn't really buy us anything but that's what was there before.